### DISCUSSION TITLE
Can't login with GitHub account: "Account not found"

### DISCUSSION THREAD

**Discussion by HeitorSF5** [vercel/vercel#9221] [Help] [answered]
I've tried to login into vercel.com with my correct GitHub credentials and I get a message in red saying "Account not found". I have already checked if Vercel is authorized as a 3rd party in my GitHub settings and everything seems nominal.

**Comment by amyegan** [comment, marked as answer]
Sorry to hear you're having this problem @HeitorSF5. 🙁 Please send a message to Vercel Support at [registration@vercel.com](mailto:registration@vercel.com). Be sure to include information about when the problem started happening, the steps you have tried already, and when you originally created the account.

**Reply by amyegan** [reply]
You may be able to [log in using your email address](https://vercel.com/login/email) instead of using the _Continue with GitHub_ button.

https://vercel.com/login/email

**Comment by the-yex** [comment]
+1
